packages/trpc/server/routers/viewer/slots/util.ts

```diff
@@ -62,33 +62,38 @@ export const checkIfIsAvailable = ({
     return true;
   }
 
-  const slotEndTime = time.add(eventLength, "minutes").utc();
-  const slotStartTime = time.utc();
+  const slotStartValue = time.utc().valueOf();
+  const slotEndValue = time.add(eventLength, "minutes").utc().valueOf();
 
   return busy.every((busyTime) => {
-    const startTime = dayjs.utc(busyTime.start).utc();
-    const endTime = dayjs.utc(busyTime.end);
+    const busyStartValue = dayjs.utc(busyTime.start).valueOf();
```

Comment:
Using toDate is preferred here, it's auto-cast to valueOf()

Reply:
done